subcommittees/mentoring/minutes/minutes-2017-07-31.md

```diff
@@ -0,0 +1,182 @@
+# Software Carpentry Mentoring Subcommittee Meeting
+
+Date: 2017-07-31
+
+Time: 13:00 & 21:00 UTC
+
+## Attending
+
+- Jamie Hadwin (1st meeting)
+- Toby Hodges (1st)
+- Paula Andrea Martinez (1st)
+- Marian Schmidt (2nd)
+- Chris Hamm (2nd)
+- Mark Laufersweiler (2nd)
+
+## Apologies
+
+- Belinda Weaver
+
+## Actions/Resolutions
+
+- TH to draft a blog post on the benefits of getting involved in discussion sessions, send to JH & MS
+- JH to share checklist document for discussion session coordinator activities withh PA, then publicly when it's more polished
+
+## Minutes
+
+### Welcome
+
+#### Introductions
+
+### Recap from last meeting
+
+#### Discussion sessions
+
+- Still actively recruiting hosts/co-hosts each month - would like to see some new faces :)
+- Instructors and helpers are encouraged to partcipate in discussions
+- Juan has begun to do pre/post workshop "web meets" to get instructors/helpers in a video call mindset!
+
+#### Task force
+
+- Mentoring Committee Onboarding doc: https://tinyurl.com/onboarding-mentoring
+- This document also includes an instructor discussion host and co-host expectations and checklist:
+  - Modify document to include “Mentoring mentors” → those who host discussions can help to mentor those who are first time notetakers or assistant/co-hosts for the discussion session.
+
+#### Trainer updates (Erin Becker): 
+
+- New cohort of Trainers starting Training Program in July. Will be completing in September. 
+- At least 2 open instructor training events in July. Both in Europe and Africa. Will be seeing new trainees from new locations going through discussions in the next few months.
+
+#### Steering committee updates (Christina Koch): 
+
+- New community lead is Belinda!  :D  And she starts soon, if not already.
```

Comment:
This is in the "Recap from last meeting" section. I'll remove the second sentence, to avoid further confusion.